### ISSUE TITLE
Typo in Neeo SDK docs

### ISSUE THREAD

**Issue by gvdhoven** [NEEOInc/neeo-sdk#186]
[startServer(configuration)](https://neeoinc.github.io/neeo-sdk/#src-lib-index.ts-startserver) contains configuration.name which has as explanation "device name" which should be "server name" right?

**Comment by pfiaux** [comment]
Correct, that's probably left over from when the SDK only ran 1 driver, it should be server name.
